ingress-examples/autoscale/README.md

```diff
@@ -0,0 +1,151 @@
+
+    ```json
+    ➜ curl -s https://pvg-classic-od16r554ux7-1e7743ca80a399c9cff4eaf617434c72-0000.us-south.stg.containers.appdomain.cloud/ | jq .
+    {
+      "path": "/",
+      "headers": {
+        "host": "pvg-classic-od16r554ux7-1e7743ca80a399c9cff4eaf617434c72-0000.us-south.stg.containers.appdomain.cloud",
+        "x-request-id": "4780c7a0baa98c993d474be463c8b643",
+        "x-real-ip": "10.189.192.233",
+        "x-forwarded-for": "10.189.192.233",
+        "x-forwarded-host": "pvg-classic-od16r554ux7-1e7743ca80a399c9cff4eaf617434c72-0000.us-south.stg.containers.appdomain.cloud",
+        "x-forwarded-port": "443",
+        "x-forwarded-proto": "https",
+        "x-forwarded-scheme": "https",
+        "x-scheme": "https",
+        "user-agent": "curl/7.88.1",
+        "accept": "*/*"
+      },
+      "method": "GET",
+      "body": "",
+      "fresh": false,
+      "hostname": "pvg-classic-od16r554ux7-1e7743ca80a399c9cff4eaf617434c72-0000.us-south.stg.containers.appdomain.cloud",
+      "ip": "10.189.192.233",
+      "ips": [
+        "10.189.192.233"
+      ],
+      "protocol": "https",
+      "query": {},
+      "subdomains": [
+        "containers",
+        "stg",
+        "us-south",
+        "pvg-classic-od16r554ux7-1e7743ca80a399c9cff4eaf617434c72-0000"
+      ],
+      "xhr": false,
+      "os": {
+        "hostname": "alb-autoscale-example-deployment-56875c56f5-njm8h"
+      },
+      "connection": {}
+    }
+    ```
+
+3. After a few minutes, you can use the following commands to see the new custom metric: `kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/alb-autoscale-example/ingress/alb-autoscale-example-ingress/nginx_ingress_controller_requests_rate" | jq .`
+
+    ```bash
+    $ kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/alb-autoscale-example/ingress/alb-autoscale-example-ingress/nginx_ingress_controller_requests_rate"
+    {
+      "kind": "MetricValueList",
+      "apiVersion": "custom.metrics.k8s.io/v1beta2",
+      "metadata": {},
+      "items": [
+        {
+          "describedObject": {
+            "kind": "Ingress",
+            "namespace": "alb-autoscale-example",
+            "name": "alb-autoscale-example-ingress",
+            "apiVersion": "networking.k8s.io/v1"
+          },
+          "metric": {
+            "name": "nginx_ingress_controller_requests_rate",
+            "selector": null
+          },
+          "timestamp": "2023-08-08T09:11:02Z",
+          "value": "0"
+        }
+      ]
+    }
+    ```
+
+    * When the `kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/alb-autoscale-example/ingress/alb-autoscale-example-ingress/nginx_ingress_controller_requests_rate" | jq .` command is not working after 10 minutes, please review the `apiservice v1beta2.custom.metrics.k8s.io` apiservice, the `AVAILABLE` must be `true`, otherwise check the `alb-prometheus-adapter` service and `alb-prometheus-adapter` in `alb-autoscale-example` namespace:
```

Comment:
```suggestion
    * If the above command fails even after 10 minutes, review the `apiservice v1beta2.custom.metrics.k8s.io` APIService: `AVAILABLE` must be `True`, otherwise you will need to check the `alb-prometheus-adapter` Service and `alb-prometheus-adapter` deployment in `alb-autoscale-example` namespace:
```